plugins/Satellites/src/gui/SatellitesDialog.cpp

```diff
@@ -1586,6 +1587,16 @@ void SatellitesDialog::savePredictedIridiumFlares()
 							QDir::homePath() + "/iridium_flares.xlsx",
 							filter,
 							&defaultFilter);
+	#else
```

Comment:
This should not be an exclusive else: If XLSX is supported, allow export of XLSX and also CSV. 
(OK, I know, this is unimportant, currently parked code...)